regex-parsing/src/test/java/org/sonarsource/analyzer/commons/regex/python/PythonRegexSourceTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * SonarSource Analyzers Regex Parsing Commons
+ * Copyright (C) 2009-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.analyzer.commons.regex.python;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.sonarsource.analyzer.commons.regex.CharacterParser;
+import org.sonarsource.analyzer.commons.regex.RegexFeature;
+
+class PythonRegexSourceTest {
+
+  @Test
+  void testFeatures() {
+    var source = new PythonRegexSource("") {
+      @Override
+      public CharacterParser createCharacterParser() {
+        return null;
+      }
+    };
+
+    var features = source.features();
+    Assertions.assertThat(features).hasSize(7);
+    Assertions.assertThat(features).containsOnly(
```

Comment:
You can probably update the import to `import static org.assertj.core.api.Assertions.assertThat;` and chain the two assertions to make this a bit more concise.